### DISCUSSION TITLE
Lets talk and discuss about Grule

### DISCUSSION THREAD

**Discussion by newm4n** [hyperjumptech/grule-rule-engine#251] [General]
@jinagamvasubabu @niallnsec  , Hi man.

Lets talk about Grule. I will explain anything that I know about it. 
How the ANTLR4's AST model works in Grule.
How the RETE were implemented in Grule.

This invitation is open for every body interested in Grule.

I'm living in Indonesia, GMT+7. Lest find the best time to meet or zoom.

**Comment by jinagamvasubabu** [comment]
Hi @newm4n, thank you.

I will create a zoom meeting(public) for the same. 


@niallnsec - can you ping your personal email id ?

**Reply by newm4n** [reply]
Lets use https://www.timeanddate.com/worldclock/meeting.html to find the best time. What city you're located @jinagamvasubabu  @niallnsec

**Reply by niallnsec** [reply]
London would be the closest to me

**Reply by jinagamvasubabu** [reply]
Bangalore (India) is closest place to me

**Reply by newm4n** [reply]
@niallnsec @jinagamvasubabu , Im ok for Friday through Sunday. How about Saturday ?

![image](https://user-images.githubusercontent.com/3471399/136170837-3b9460dd-f51a-4c26-bb6b-cc8d2260f168.png)

**Reply by niallnsec** [reply]
Saturday 10:00 or 11:00 would be good for me

**Reply by jinagamvasubabu** [reply]
looks good to me, i am available on saturday( proposed times)

**Reply by newm4n** [reply]
Ok. 

Saturday. October 9th . 
16:00 - Indonesia.
14:30 - India.
10:00 - UK.

Let me create the invitation for you guys.

**Reply by jinagamvasubabu** [reply]
many thanks @newm4n. my email id is `jinagamvasubabu@gmail.com`

**Comment by niallnsec** [comment]
Hi,

Thanks for suggesting this.

I'm living in the UK, so currently GMT+1.

I won't be able to do anything today, but either later in the week or early next week I could join.

Email: niall.newman (at) icloud (dot) com

**Comment by newm4n** [comment]
Invitation sent. Please confirm you've received it.
Feel free to forward the invitation to those who might interested to join.
For others who follow this discussion, please email me if you wish me to forward you guys the Zoom invitation.  (you can find my email in my profile page)

**Reply by jinagamvasubabu** [reply]
Thanks, I have received it

**Comment by jinagamvasubabu** [comment]
@niallnsec are you going to join ? or you wanted us to postpone it ?

**Comment by niallnsec** [comment]
Really sorry, I got my second shot of the COVID vaccine yesterday afternoon and its hit me like a brick. First one was fine so I wasn't expecting it. If we could rearrange that would be great, if not though thats okay.

**Reply by jinagamvasubabu** [reply]
No probs, we will be having more and more sessions in near future. Get well soon and take rest

**Reply by newm4n** [reply]
Don't worry man. My second shot also a big bang. We will talk again soon. Take a good rest and you'll be fine.

**Comment by jinagamvasubabu** [comment]
@newm4n thanks a lot for the discussion. I really appreciate that.
